README.md

```diff
@@ -42,94 +58,43 @@ Below is a sample playbook:
-tenant_id | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. |
-token/tenant_password | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. In latest Deep Security Manager, "tenant_password" has been replaced with "token". "tenant_password" has been kept for backwards compatibility |
-policy_id | String | The Deep Security ID assigned to the policy and applied to the agents on activation. |
-group_id | String | The Deep Security ID assigned to the computer group and applied to the agents on activation. |
-force_reactivation | Boolean | Force re-activation even though the Deep Security Agent has already been activated. | false
-
-
-#### Task : manage.yml
-###### action:
-
-* check-in-with-manager
-* create-diagnostic-package
-* create-integrity-baseline
-* run-recommendation-scans
-* scan-for-integrity-changes
-* scan-for-malware
-
-The <code>manage</code> task lets you operate the Deep Security Agent without requiring you to log in to the Deep Security Manager. All of the available actions assume that the Deep Security Agent is installed and activated properly. Most of the actions can be triggered without any additional variables. Only <code>set-policy</code> and <code>set-policyid</code> need a corresponding policy name or id.
-
-###### action: set-policy-by-name
-
-Key | Type | Description | Default
-----|------|-------------|--------
-policy_name | String | The name of the Deep Security policy | 
-
-###### action: set-policy-by-id
+dsm_agent_download_port | Integer | The port to connect to the Deep Security Manager to download the agents. This is typically the same port as the one used to access the Deep Security Manager administration interface. | 443
+force_reactivation | Boolean | Force re-activation even if the Deep Security Agent has already been activated. | false
+group_id | String | The Deep Security ID assigned to the computer group and applied to the agents on activation. | 1
+policy_id | String | ID of the Deep Security Policy to be assigned to the agents. | 1
+policy_name | String | Name of the Deep Security Policy to be assigned to the agents. | Base Policy
+tenant_id | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. <br /> **Only required in multi-tenancy environment** | 111A111A-1A1A-11AA-AAA-11AA11111111
+token/tenant_password | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. In latest Deep Security Manager, "tenant_password" has been replaced with "token". "tenant_password" has been kept for backwards compatibility.<br /> **Only required in multi-tenancy environment** | 111A111A-1A1A-11AA-AAA-11AA11111111
```

Comment:
Only required in multi-tenancy environment: -> Multi-tenancy only: (to be consistent)